machine_data_model/nodes/connectors/opcua_connector.py

```
@@ -0,0 +1,440 @@
+    if policy_string == "SecurityPolicyBasic256Sha256":
+        policy = SecurityPolicyBasic256Sha256
+
+    return policy
+
+
+class OpcuaConnector(AbstractConnector):
+    """
+    Represents an OPCUA client
+    """
+
+    def __init__(
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)